website/docs/components/dropdown/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,211 @@
+## When to use
+
+- To display a list of actions under a single button toggle.
+
+## When not to use
+
+- In forms. When providing more than 5-7 options in a form, use a [Select](/components/form/select/overview) instead.
+- When selecting an option results in immediate navigation or update to the page content, use a Context Switcher _(coming soon)_.
+
+## ToggleButton Size
+
+ToggleButtons come in two sizes: **small** and **medium**. This allows for placement in ButtonSets with buttons of the same size.
+
+### Small
+
+<Hds::ButtonSet>
+  <Hds::Dropdown as |dd|>
+    <dd.ToggleButton @text="Primary" @size="small"/>
+    <dd.Interactive @text="Item One" />
+    <dd.Interactive @text="Item Two" />
+    <dd.Interactive @text="Item Three" />
+    <dd.Interactive @text="Item Four" />
+  </Hds::Dropdown>
+  <Hds::Dropdown as |dd|>
+    <dd.ToggleButton @text="Secondary" @color="secondary" @size="small" />
+    <dd.Interactive @text="Item One" />
+    <dd.Interactive @text="Item Two" />
+    <dd.Interactive @text="Item Three" />
+    <dd.Interactive @text="Item Four" />
+  </Hds::Dropdown>
+</Hds::ButtonSet>
+
+### Medium
+
+<Hds::ButtonSet>
+  <Hds::Dropdown as |dd|>
+    <dd.ToggleButton @text="Primary" />
+    <dd.Interactive @text="Item One" />
+    <dd.Interactive @text="Item Two" />
+    <dd.Interactive @text="Item Three" />
+    <dd.Interactive @text="Item Four" />
+  </Hds::Dropdown>
+  <Hds::Dropdown as |dd|>
+    <dd.ToggleButton @text="Secondary" @color="secondary" />
+    <dd.Interactive @text="Item One" />
+    <dd.Interactive @text="Item Two" />
+    <dd.Interactive @text="Item Three" />
+    <dd.Interactive @text="Item Four" />
+  </Hds::Dropdown>
+</Hds::ButtonSet>
+
+## Chevron
+
+Open toggles use a chevron pointing up, while closed toggles use a chevron pointing down.
+
+!!! Info
+
+**Figma Tip**
+
+Set `isOpen=true` when displaying the toggle with a menu and `isOpen=false` when displaying the toggle.
+
+!!!
+
+### Open
+
+<Hds::Dropdown as |dd|>
+  <dd.ToggleButton @text="Manage" @color="secondary" @isOpen={{true}} />
+  <dd.Interactive @text="Rename" />
+  <dd.Interactive @text="Edit" />
+  <dd.Interactive @text="Restore" />
+</Hds::Dropdown>
+
+### Closed
+
+<Hds::Dropdown as |dd|>
+  <dd.ToggleButton @text="Manage" @color="secondary" />
+  <dd.Interactive @text="Rename" />
+  <dd.Interactive @text="Edit" />
+  <dd.Interactive @text="Restore" />
+</Hds::Dropdown>
+
+### ToggleIcon
+
+Chevrons indicate that ToggleIcon opens the list, so they’re required for all variations. The only (temporary) exception is the overflow menu, which uses the more icon.
+
```

Comment:
I would probably put the name of the icon in quotes for clarity.

IE: "uses the “more” icon"